stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -1124,15 +1280,10 @@ extension Unsafe${Mutable}BufferPointer {
   ///     method.
   ///   - buffer: The buffer temporarily bound to `T`.
   /// - Returns: The return value, if any, of the `body` closure parameter.
-  @inlinable // unsafe-performance
   @_alwaysEmitIntoClient
-  // This custom silgen name is chosen to not interfere with the old ABI
-% if Mutable:
-  @_silgen_name("_swift_se0333_UnsafeMutableBufferPointer_withMemoryRebound")
-% else:
-  @_silgen_name("_swift_se0333_UnsafeBufferPointer_withMemoryRebound")
-% end
-  public func withMemoryRebound<T, Result>(
+  // FIXME(NCG): Get rid of this once the new mangling rules are working.
```

Comment:
Outdated?

Reply:
Yep, I removed the `@_silgen_name`. We could also rename the obsolete function back to its original name and remove the silgen name wart from there too, but I'll leave that exercise for later.